pkg/oc/cli/describe/printer.go

```diff
@@ -858,6 +858,14 @@ func printRoleList(list *authorizationapi.RoleList, w io.Writer, opts kprinters.
 	return nil
 }
 
+func truncatedList(list []string, maxLength int) string {
+	result := strings.Join(list, ", ")
+	if len(list) > maxLength {
+		result = strings.Join(list[0:maxLength], ", ") + fmt.Sprintf(" (%d more)", len(list)-maxLength)
```

Comment:
I find this more readable and efficient
`fmt.Sprintf("%s (%d more)", strings.Join(list[0:maxLength], ", "), len(list)-maxLength)`

Reply:
yeah, it can be optimized even more ;-) fixed.